### DISCUSSION TITLE
How u customized your profile?

### DISCUSSION THREAD

**Discussion by Quantum-Codes** [gosoccerboy5/gosoccerboy5#8] [General]
I am Ankit_Anmol on scratch (hopefully u know me xD) 

How did u customize ur profile?
Edit: ohhh I figured it out :)

**Comment by gosoccerboy5** [comment]
alright can I close this?

**Reply by Quantum-Codes** [reply]
yes :) hru?

**Reply by Quantum-Codes** [reply]
What r gists?

**Reply by gosoccerboy5** [reply]
They're small snippets of code you can share and embed. They're essentially miniature Git repositories. https://en.wikipedia.org/wiki/GitHub#Gist

**Reply by Quantum-Codes** [reply]
Oh ok

**Comment by Quantum-Codes** [comment]
Github.com = cat.
Scratch.mit.edu = cat.
Why? Do you know? (Just an interesting observation)

**Reply by gosoccerboy5** [reply]
it's probably just a coincidence
